wp-content/themes/classicpress-susty-child/template-parts/content.php

```diff
@@ -10,17 +10,6 @@
 ?>
 
 <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
-	<header class="blog">
-		<?php
-		if ( is_singular() ) :
-			the_title( '<h1>', '</h1>' );
-		else :
-			the_title( '<h2><a href="' . esc_url( get_permalink() ) . '" rel="bookmark">', '</a></h2>' );
```

Comment:
The `<h2>` titles are used on some of the archive pages like post category listings: https://www.classicpress.net/blog/category/classicpress-values/

I am going to change this to only output the `<header class="blog">` with `<h2>` title if `! is_singular()`.

Reply:
Good catch. Fine with me.